### PR TITLE
Cleanup and simplify EL builds.

### PR DESCRIPTION
**Pull request by zmarano**
- Remove EL6 builds.
- Remove MBR builds which are deprecated.
- Revamp EL7 kickstart configs to get rid of old EL6 constructs and cleanup to mimic EL8 config.
- EL7 uses chrony instead of ntpd - remove ntpd.
- Add CentOS Stream 8 config.
- Simplify kickstart file generator.

### CODE REVIEW

File: daisy_workflows/build-publish/enterprise_linux/centos_stream_8.wf.json [hunk 1]
```diff
@@ -22,18 +22,18 @@
     },
     "installer_iso": {
       "Required": true,
-      "Description": "The RHEL 6 installer ISO to build from."
+      "Description": "The CentOS 7 installer ISO to build from."
```
Comment (gaohannk) [inline daisy_workflows/build-publish/enterprise_linux/centos_stream_8.wf.json:25]: Centos Stream 8

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel-7-options.cfg [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# el8-options.cfg
+# rhel7-options.cfg
```
Comment (gaohannk) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel-7-options.cfg:1]: rhel-7-options.cfg

File: daisy_workflows/build-publish/enterprise_linux/centos_stream_8.wf.json (hunk 1 above)
Comment (zmarano) [reply daisy_workflows/build-publish/enterprise_linux/centos_stream_8.wf.json]: Done

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel-7-options.cfg (hunk 2 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel-7-options.cfg:1]: done

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel-8-options.cfg [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# el8-uefi-options.cfg
+# rhel8-options.cfg
```
[the remark this answers is not in the document]
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel-8-options.cfg:1]: done

**Review by EricEdens — APPROVED** [review]

**Pull request merged**